naurffxiv/src/app/ultimates/dsr.js

```diff
@@ -0,0 +1,226 @@
+import react from "react";
+
+export default function DSR() {
+
+    return (
+        <>
+            <div className="ultimate-header">
+                <h1>Dragonsong's Reprise (Ultimate)</h1>
+            </div>
+
+            <div className="table-of-contents">
+                <h2>Strats by Phase</h2>
+                <p>All strats found here are often referred to as APD (Aether PUG Discord), a predecessor to NAUR.</p>
+                <h3>Phase 1: Vault Knights</h3>
+                <ul>
+                    <li><a href="https://ff14.toolboxgaming.space/?id=802461081524561&preview=1#1" target="_blank">Vault Knights Toolbox</a></li>

```

Comment:
so im not really sure why this is the case but it seems like theres no blue highlighting on the web page. maybe we'll have to take a look at the css and see why it's being overwritten

Reply:
I assume it's from the tailwind css, I didn't really plan to do any styling in this pr

Reply:
ok sgtm